src/Server/Query.elm

```diff
@@ -92,13 +94,12 @@ materialParser key =
             )
 
 
-countriesParser : String -> Query.Parser (List Country.Code)
-countriesParser key =
+countryParser : String -> Query.Parser Country.Code
+countryParser key =
     Query.string key
         |> Query.map
-            (Maybe.map
-                (String.split "," >> List.map Country.Code)
-                >> Maybe.withDefault []
+            (Maybe.map Country.codeFromString
+                >> Maybe.withDefault (Country.codeFromString "")
```

Comment:
Yeah, I don't know, maybe we could "invent" a country code materlalizing an invalid code? eg `"XX"`? The Parser.Query Api doesn't allow us to use Results, sadly… But we may want to spend a little time investigating if it could be doable, because that would allow us to validate the query string parameters! That would be super valuable, we could even release some OSS lib because I suspect this is a common need.